R/zchunk_L2242.land_input_4_irr_mgmt.R

```diff
@@ -1,90 +1,120 @@
 #' module_aglu_L2242.land_input_4_irr_mgmt
 #'
-#' Briefly describe what this chunk does.
+#' This chunk generates files of the fourth land nest by region.
```

Comment:
What does "fourth land nest" mean? Can we provide something clearer or more specific?

Reply:
Maybe note what sort of data is in the file (land allocation, carbon) and what type of land (nodes/leafs, crops)